src/System.Text.Encoding/tests/UTF32Encoding/UTF32EncodingDecode.cs

```diff
@@ -102,27 +102,30 @@ public void Decode_InvalidBytes()
             Decode_InvalidBytes(new byte[] { 0xFF, 0xFF, 0xFF, 0xFF }, 0, 4, "\uFFFD");
         }
 
-        public static byte[] GetBigEndianBytes(byte[] littleEndianBytes)
+        public static byte[] GetBigEndianBytes(byte[] littleEndianBytes, int index, int count)
```

Comment:
Thanks - I had to tweak the implementation a little bit, as the new byte array has to be the same length as the `litteEndianBytes`, not `count`